torch/_torch_docs.py

```diff
@@ -4624,10 +4624,15 @@ def merge_dicts(*dicts):
 rsqrt(input, out=None) -> Tensor
 
 Returns a new tensor with the reciprocal of the square-root of each of
-the elements of :attr:`input`.
+the elements of ``input``.
 
 .. math::
     \text{out}_{i} = \frac{1}{\sqrt{\text{input}_{i}}}
+
+Note that this function, whenever possible, attempts to use an algorithm that
+is faster but guarantees less accuracy than that of ``1/torch.sqrt(input)``. If
```

Comment:
nit: I think it's clearer if you say "it does not guarantee as much accuracy" -- there are almost certainly cases where the the accuracy is the same.